integration/container/create_test.go

```diff
@@ -301,3 +302,79 @@ func TestCreateWithCustomReadonlyPaths(t *testing.T) {
 		checkInspect(t, ctx, name, tc.expected)
 	}
 }
+
+func TestCreateWithInvalidHealthcheckParams(t *testing.T) {
+	skip.If(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.32"), "broken in earlier versions")
```

Comment:
Looks like earlier versions were "functional" but returned a 500 status instead of a 4xx; if we add back checks for status code, can we also re-enable this for API < 1.32 ?

Reply:
Thank you for the review. Updated the test to include API < 1.32.